src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+/**
+ * Represents the text parser for Duke. When parsing a user input, the parser
+ * also handles the updating of the associated TaskList.
+ *
+ * @author Joshua Yong
+ */
+public class Parser {
+
+    // The TaskList which the Parser updates
```

Comment:
Perhaps you could change this comment about this class member to a single-line Javadoc like so: 
```suggestion
    /** The TaskList which the Parser updates */
```